core/src/test/java/org/javarosa/core/model/test/GeoPointTests.java

```diff
@@ -0,0 +1,52 @@
+package org.javarosa.core.model.test;
+
+import org.javarosa.core.model.FormIndex;
+import org.javarosa.core.model.condition.EvaluationContext;
+import org.javarosa.core.test.FormParseInit;
+import org.javarosa.form.api.FormEntryController;
+import org.javarosa.test_utils.ExprEvalUtils;
+import org.javarosa.xpath.parser.XPathSyntaxException;
+import org.junit.Before;
+import org.junit.Test;
+
+public class GeoPointTests {
+
+    private FormParseInit geopointFpi;
+    private EvaluationContext geopointEvalCtx;
+
+    @Before
+    public void setUp() {
+        geopointFpi = new FormParseInit("/geopoint_tests.xml");
+
+        FormEntryController fec = geopointFpi.getFormEntryController();
+        fec.jumpToIndex(FormIndex.createBeginningOfFormIndex());
+
+        // step through entire form
+        do {
+        } while (fec.stepToNextEvent() != FormEntryController.EVENT_END_OF_FORM);
```

Comment:
You can actually remove lines 21 - 26 since none of the tests actually make use of more complex form logic.